app/src/renderer/components/common/TmPage.vue

```diff
@@ -14,7 +14,8 @@
       <slot slot="header-buttons" name="header-buttons" />
     </tm-page-header>
     <main class="tm-page-main">
-      <template v-if="this.$slots['managed-body']">
+      <card-sign-in-required v-if="signinRequired && !session.signedIn" />
```

Comment:
should this go inside of the template below with connecting, loading, and error?

Reply:
either way. I can do it.